modules/pam_unix/passverify.c

```diff
@@ -858,14 +859,14 @@ PAMH_ARG_DECL(int unix_update_passwd,
       freecon(passwd_context_raw);
     }
 #endif
-    pwfile = fopen(PW_TMPFILE, "w");
+    pwfile = fopen(PW_TMPFILE, "we");
```

Comment:
"e" required glibc >= 2.7, which is probably OK nowadays, but let's note this in the commit message.